tools/c7n_azure/c7n_azure/provisioning/function_app.py

```diff
@@ -46,16 +54,10 @@ def _provision(self, params):
         # general app settings
         con_string = params['storage_account_connection_string']
         site_config.app_settings.append(azure_name_value_pair('AzureWebJobsStorage', con_string))
-        site_config.app_settings.append(azure_name_value_pair('AzureWebJobsDashboard', con_string))
         site_config.app_settings.append(azure_name_value_pair('FUNCTIONS_EXTENSION_VERSION',
                                                               FUNCTION_EXT_VERSION))
         site_config.app_settings.append(azure_name_value_pair('FUNCTIONS_WORKER_RUNTIME', 'python'))

```

Comment:
Please add PYTHONDONTWRITEBYTECODE=1 here (to disable pycache generation which restarts Functions host)